modules/ROOT/pages/audit-logging.adoc

```diff
@@ -29,7 +31,18 @@ Each log entry has a set of properties that provides information about the activ
 * *Type*: The type of the object on which the action is performed, for example, `Organization`
 * *Action*: The action associated with the object, for example, `Create`
 * *Object*: The name of the Object, for example, `foo`
-* *User*: The User who performed the action, for example, `johndoe`
+* *User*: The User who performed the action, for example, `johndoe` 
++
+[NOTE]
+--
+You may occasionally see an *Anypoint Staff* log entry. This log entry is made when the audit action was performed by a user who does not belong to your master organization. +
```

Comment:
may is a translation problem, how about Some log entries are marked as `Anypoint Staff` (this is a log entry, not a UI label, yes?).